trunk/ii02823/task_01/src/task_01.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+#include <cmath>
+
+using namespace std;
```

Comment:
If you'll write "using namespace std" it can lead to naming conflicts. 
You have 2 options to improve the code:
1) use std::cout, std::cin, std::stirng etc
2) use the following declarations:
```suggestion
using std::cout;
using std::cin;
```